tools/export/blender25/godot_export_manager.py

```diff
@@ -323,17 +348,19 @@ def execute(self,context):
                 object.select = False
             context.scene.objects.active = None
 
-            ### make particle duplicates, parent and select them
+            # Make particle duplicates, parent and select them
             nodes_to_be_added = []
             if group[self.idx].use_include_particle_duplicates:
-                for i,object in enumerate(group[self.idx].nodes):
+                for i, object in enumerate(group[self.idx].nodes):
                     if bpy.data.objects[object.name].type != "EMPTY":
-                        context.scene.objects.active = bpy.data.objects[object.name]
+                        context.scene.objects.active = bpy.data.objects[
+                            object.name]
                         bpy.data.objects[object.name].select = True
                         bpy.ops.object.duplicates_make_real()
                         for object in context.selected_objects:
                             nodes_to_be_added.append(object)
-                        bpy.ops.object.parent_set(type="OBJECT", keep_transform=False)
+                        bpy.ops.object.parent_set(type="OBJECT",
+                                                  keep_transform=False)
```

Comment:
Same here; does PEP8 really enforce cropping at 78-80 chars mechanically, or is it the tool you used that is a bit too drastic? In such cases I think common sense would be to not to crop if keeping a full statement on a single line means being between 80-85 chars. I'm no python guru, but that's the rule I apply visually in my own scripts :)